pre-commit/action.yaml

```diff
@@ -18,10 +21,25 @@ runs:
       with:
         token: ${{ inputs.token }}
 
+    - name: Set target files
+      id: set-target-files
+      run: |
+        if [ -z "${{ inputs.base-branch }}" ]; then
+          echo "option=--all-files" >> $GITHUB_OUTPUT
+          exit 0
+        fi
+        echo "option=--from-ref ${{ inputs.base-branch }} --to-ref HEAD" >> $GITHUB_OUTPUT
+      shell: bash
+
+    - name: Get target files
```

Comment:
I think this step can be removed because it's shown in the next step.

Reply:
Removed Updated https://github.com/autowarefoundation/autoware-github-actions/pull/237/commits/00e452a0e130ab2b5331cba8aeecda736151c269.